src/modules/flow/http-client/http-client.json

```diff
@@ -0,0 +1,267 @@
+{
+  "$schema": "http://solettaproject.github.io/soletta/schemas/node-type-genspec.schema",
+  "name": "http-client",
+  "meta": {
+    "author": "Intel Corporation",
+    "license": "BSD-3-Clause",
+    "version": "1"
+  },
+  "types": [
+    {
+      "name": "http-client/boolean",
+      "category": "output/network",
+      "description": "HTTP Client for boolean",
+      "methods": {
+        "close": "common_close",
+        "open": "common_open"
+      },
+      "node_type": {
+        "access": [
+          "base"
+        ],
+        "data_type": "struct http_client_node_type",
+        "extra_methods": {
+	  "process_token": "boolean_process_token",
+	  "process_data": "boolean_process_data"
+        }
+      },
+      "options": {
+	"members": [
+	  {
+	    "data_type": "string",
+	    "description": "The url that will be used",
+	    "name": "url"
+	  }
+	],
+	"version": 1
+      },
+      "in_ports": [
+        {
+          "data_type": "string",
+          "description": "Url wich request will be done",
```

Comment:
s/wich/which/
Btw, there are places where url description is 'The url that will be used', maybe is good to use the same string on all places (I'd rather the second one, or something even more descriptive).